src/main/java/sorald/rule/RuleType.java

```diff
@@ -1,9 +1,20 @@
 package sorald.rule;
 
 /** Enumeration of Sonar rule types */
-public enum RuleType {
-    BUG,
-    VULNERABILITY,
-    CODE_SMELL,
-    SECURITY_HOTSPOT;
+public enum RuleType implements IRuleType {
```

Comment:
```suggestion
public enum SonarRuleType implements IRuleType {
```
We can be more explicit here, I think.

Reply:
We can also move `SonarRuleType` inside `sonar` package. Eventually, we can make `sonar` package a dependent module on the SPI we are working on.

Reply:
> Eventually, we can make sonar package a dependent module on the SPI we are working on.

Refactoring the sonar package to a subproject is the long-term goal.